lib/key-generators/git-commit.js

```diff
@@ -0,0 +1,23 @@
+var CoreObject  = require('core-object');
+var gitRepoInfo = require('git-repo-info');
+var Promise     = require('ember-cli/lib/ext/promise');
+
+module.exports = CoreObject.extend({
+  generate: function() {
+    var path = gitRepoInfo._findRepo();
```

Comment:
Just want to confirm that this will be finding the path to the git repo of the consuming project.  I'm just asking this because I know I came across an issue (and I can't remember the exact details), when I was using git commits hashes initially in ember-cli-deploy. From memory, I think it was when I had npm linked an addon, the repo that was being inspected was actually that of the addon and not the consuming project.

I haven't used `git-repo-info` so maybe this is cool.  Just wanted to call it out to make sure.